Client/Frontend/Fakespot/Components/FakespotBottomSheetChildViewController.swift

```diff
@@ -0,0 +1,106 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Common
+import ComponentLibrary
+import UIKit
+
+class FakespotBottomSheetChildViewController: UIViewController, BottomSheetChild, Themeable {
```

Comment:
Is there a reason to keep this around? It doesn't seem to be used.

Reply:
It is very strange that in the Components folder I don't have anything...